### PR TITLE
add public repo note in CI docs

### PR DESCRIPTION
**Pull request by SaraVieira**
Adds a note on how CI only works in private repos

### CODE REVIEW

File: packages/homepage/content/docs/8-ci.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ your library by creating a file called `ci.json` in a folder called
 `.codesandbox` (`.codesandbox/ci.json`) in the root of your repo. An example PR
 can be found [here](https://github.com/facebook/react/pull/17175).
 
+**Note:** CodeSandbox CI only works in public repositories for now.
```
Comment (peduarte) [inline packages/homepage/content/docs/8-ci.md:44]: 🙌

**Review by peduarte — APPROVED** [review]

**Comment by CompuIves** [comment]
Would be good to add a note to send us an email if you're interested in private repo support! Gives us a better idea on how quickly we should pick this up.

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5021/commits/33b753c7c233d0bfa667b078cf478147230f161b is at https://pr5021.build.csb.dev/s/new.

**Review by CompuIves — APPROVED** [review]

**Comment by SaraVieira** [comment]
Will add it but merged it for now since I just left my house for food

**Pull request merged**